org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -0,0 +1,453 @@
+            loudnesschange,
+            getLastChangedResource(stateReader, 'loudness', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('motion')) {
+        var motion = createResource('oic.r.sensor.motion', 'oic.if.s', 'motion', expand, {
+            value: stateReader.get('motion')
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.sensor.motion', [
+            motion,
+            getLastChangedResource(stateReader, 'motion', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('occupied')) {
+        var presence = createResource('oic.r.sensor.presence', 'oic.if.s', 'presence', expand, {
+            value: stateReader.get('occupied')
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.sensor.presence', [
+            presence,
+            getLastChangedResource(stateReader, 'occupied', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('temperature')) {
+        var temperature = createResource('oic.r.temperature', 'oic.if.s', 'temperature', expand, {
+            temperature: stateReader.get('temperature'),
+            units: "C"
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.sensor.temperature', [
+            temperature,
+            getLastChangedResource(stateReader, 'temperature', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('vibration')) {
+        var vibrationchange = createResource('oic.r.sensor', 'oic.if.s', 'vibrationchange', expand, {
+            value: stateReader.get('vibration')
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.sensor.vibrationchange', [
+            vibrationchange,
+            getLastChangedResource(stateReader, 'vibration', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('liquid_detected')) {
+        var water = createResource('oic.r.sensor.water', 'oic.if.s', 'water', expand, {
+            value: stateReader.get('liquid_detected')
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.sensor.water', [
+            water,
+            getLastChangedResource(stateReader, 'liquid_detected', expand)
+        ]));
+    }
+
+    if (stateReader.containsKey('battery')) {
+        var battery = createResource('oic.r.energy.battery', 'oic.if.s', 'battery', expand, {
+            charge: convertDeviceBatteryToTranslatorBattery(stateReader.get('battery'))
+        });
+
+        entities.push(createEntity(name, 'opent2t.d.battery', [
+            battery
+        ]));
+    }
+
+    return {
+        opent2t: {
+            schema: 'org.opent2t.sample.multisensor.superpopular',
+            translator: 'opent2t-translator-com-wink-sensorpod',
+            controlId: providerSchema['object_id']
+        },
+        pi: providerSchema['uuid'],
+        mnmn: defaultValueIfEmpty(providerSchema['device_manufacturer'], "Wink"),
+        mnmo: defaultValueIfEmpty(providerSchema['manufacturer_device_model'], "Sensor (Generic)"),
```

Comment:
The same pattern is used in other translators. I believe this is so that we have at least some value in the case where the provider doesn't have one.